elasticdl/python/common/save_utils.py

```diff
@@ -259,3 +259,11 @@ def restore_params_from_checkpoint(checkpoint_dir, shard_index, shard_num):
         parameters.embedding_params.update(embedding_tables)
         parameters.version = version
         return parameters
+
+    @staticmethod
+    def get_version_from_checkpoint(checkpoint_dir):
+        variable_shard_files = os.listdir(checkpoint_dir)
+        shard_file_path = os.path.join(checkpoint_dir, variable_shard_files[0])
```

Comment:
From this code, we only load the first file in variable_shard_files, what about other files?

Reply:
Each shard file has the version and the versions in shard files are the same. So, we only need to load one file to get the version.